spec.emu

```diff
@@ -136,189 +179,307 @@ location: https://github.com/tc39/proposal-decimal/
+      <p>An accessor property indicating whether *this* value represents a Decimal128 NaN value. The possible values are *true* and *false*.</p>
     </emu-clause>
     <emu-clause id="sec-decimal.prototype.isfinite">
-      <h1>Decimale128.prototype.isFinite</h1>
-      <p>A property indicating whether *this* value represents a finite value. The possible values are *true* and *false*. Decimal128 NaN values are considered finite.</p>
-    </emu-clause>
-    <emu-clause id="sec-decimal.prototype.significand">
-      <h1>Decimale128.prototype.significand</h1>
-      <p>A property that stores the significand of *this* value. Its type is BigInt. This property should be a BigInt even if *this* value is NaN or infinite.</p>
-    </emu-clause>
-    <emu-clause id="sec-decimal.prototype.exponent">
-      <h1>Decimale128.prototype.exponent</h1>
-      <p>A property that stores the exponent of *this* value. Its type is Number. It should always be an integer. This property should be a Number even if *this* value is NaN or infinite.</p>
+      <h1>get Decimal128.prototype.isFinite</h1>
+      <p>An accessor property indicating whether *this* value represents a finite Decimal128 value. The possible values are *true* and *false*.</p>
     </emu-clause>
     <emu-clause id="sec-decimal.prototype.abs">
       <h1>Decimal128.prototype.abs ( )</h1>
-      <p>Computes the absolute value of *this* value according to the rules specified in the IEEE 754 Decimal128. *this* value is not modified.</p>
+      <p>This method performs the following steps when called:</p>
+      <emu-alg>
+        1. Let _O_ be the **this** value.
+        1. Perform ? RequireInternalSlot(_O_, [[Decimal128Data]]).
+        1. Let _d_ be _O_.[[Decimal128Data]].
+        1. If _d_ is *NaN*<sub>D</sub>, return *NaN*<sub>D</sub>.
+        1. If _d_ is *+∞*<sub>D</sub>, return *+∞*<sub>D</sub>.
+        1. If _d_ is *-∞*<sub>D</sub>, return *+∞*<sub>D</sub>.
+        1. Let _v_ be _d_[0].
+        1. Let _q_ be _d_[1].
+        1. Let _n_ be « abs(_v_), _q_ ».
+        1. Let _N_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%Decimal128.prototype%"*, « [[Decimal128Data]] »).
+        1. Set _N_.[[Decimal128Data]] to _n_.
+        1. Return _N_.
+      </emu-alg>
     </emu-clause>
     <emu-clause id="sec-decimal.prototype.neg">
       <h1>Decimal128.prototype.neg ( )</h1>
-      <p>Computes the negation of *this* value.</p>
+      <p>This method performs the following steps when called:</p>
+      <emu-alg>
+        1. Let _O_ be the **this** value.
+        1. Perform ? RequireInternalSlot(_O_, [[Decimal128Data]]).
+        1. Let _d_ be _O_.[[Decimal128Data]].
+        1. If _d_ is *NaN*<sub>D</sub>, return *NaN*<sub>D</sub>.
+        1. If _d_ is *+∞*<sub>D</sub>, return *-∞*<sub>D</sub>.
+        1. If _d_ is *-∞*<sub>D</sub>, return *+∞*<sub>D</sub>.
+        1. Let _v_ be _d_[0].
+        1. Let _q_ be _d_[1].
+        1. Let _n_ be « -1 * _v_, _q_ ».
```

Comment:
```suggestion
        1. Let _n_ be « -1 &times; _v_, _q_ ».
```
ECMA-262 uses multiplication signs.

Reply:
Ah, OK. Should I use an entity or is it OK to use `×`?